eng/pipelines/templates/jobs/smoke.tests.yml

```diff
@@ -179,36 +179,42 @@ jobs:
               return $line
             }
 
-            $dependencies | Out-File $env:REQUIREMENTS
+            $dependencies | Out-File $(requirements)
 
           displayName: Override requirements with pipeline build artifact versions
-          env:
-            REQUIREMENTS: $(requirements)
 
-        - pwsh: pip install -r "$env:REQUIREMENTS" --no-deps --upgrade
+        - pwsh: |
+            $ErrorActionPreference = "Continue"
```

Comment:
Might be worth a comment somewhere in this script that explains why we need to retry like this and how long we usually expect to wait until the package is live.

Reply:
For reference we put a big comment at https://github.com/Azure/azure-sdk-for-python/blob/main/eng/common/scripts/git-branch-push.ps1#L40 for the same reason.